docker/Gemfile

```diff
@@ -3,5 +3,6 @@
 source "https://rubygems.org"
 
 gem "dependabot-common", path: "../common"
+gem "docker_registry2", path: "../docker_registry2/lib"
```

Comment:
```suggestion
```

Don't think we need this in the Gemfile, should be pulled in via `gemspec` below, and the path used here also does not exist